ontotext-yasgui-web-component/src/services/yasgui/ontotext-yasgui-service.ts

```diff
@@ -2,39 +2,40 @@ import {YasguiConfiguration} from '../../models/yasgui-configuration';
 import {VisualisationUtils} from '../utils/visualisation-utils';
 import {HtmlElementsUtil} from '../utils/html-elements-util';
 import {TranslationService} from '../translation.service';
+import {ServiceFactory} from '../service-factory';
 
-class OntotextYasguiServiceDefinition {
+export class OntotextYasguiService {
 
-  private static translationService: TranslationService;
+  private translationService: TranslationService;
 
-  constructor() {
-    OntotextYasguiServiceDefinition.translationService = TranslationService.Instance;
+  constructor(serviceFactory: ServiceFactory) {
+    this.translationService = serviceFactory.get(TranslationService);
   }
 
   postConstruct(hostElement: HTMLElement, config: YasguiConfiguration): void {
 
-    OntotextYasguiServiceDefinition.initEditorTabs(hostElement, config);
-    OntotextYasguiServiceDefinition.initControlBar(hostElement, config);
-    OntotextYasguiServiceDefinition.initResultTabs(hostElement, config);
-    OntotextYasguiServiceDefinition.initButtonsStyling(hostElement, config);
-    OntotextYasguiServiceDefinition.updateTranslation(config);
```

Comment:
Why did you make all these instance methods? They are perfectly OK to be static.

Reply:
You are right. I will revert them.